agent/grpc-middleware/rate_limit_mappings.gen.go

```diff
@@ -22,6 +22,12 @@ var rpcRateLimitSpecs = map[string]rate.OperationType{
 	"/hashicorp.consul.internal.peering.PeeringService/TrustBundleRead":          rate.OperationTypeRead,
 	"/hashicorp.consul.internal.peerstream.PeerStreamService/ExchangeSecret":     rate.OperationTypeWrite,
 	"/hashicorp.consul.internal.peerstream.PeerStreamService/StreamResources":    rate.OperationTypeRead,
+	"/hashicorp.consul.resource.ResourceService/Delete":                          rate.OperationTypeWrite,
+	"/hashicorp.consul.resource.ResourceService/List":                            rate.OperationTypeRead,
+	"/hashicorp.consul.resource.ResourceService/Read":                            rate.OperationTypeRead,
+	"/hashicorp.consul.resource.ResourceService/Watch":                           rate.OperationTypeWrite,
```

Comment:
Not for now, but at some point we should probably revisit how we apply rate-limits to the `Watch` endpoint, as it's really `len(event)` reads.

Reply:
Updated `Watch` to a read in https://github.com/hashicorp/consul/pull/16528/commits/73fee3ff8b6779124211ebfaa449c27afb3f1095